app/src/main/java/vn/sunasterisk/music_67/ui/search/SearchActivity.kt

```diff
@@ -0,0 +1,118 @@
+package vn.sunasterisk.music_67.ui.search
+
+import android.content.Intent
+import android.os.Handler
+import android.support.v7.widget.SearchView
+import android.view.View
+import android.widget.Toast
+import kotlinx.android.synthetic.main.activity_search.*
+import vn.sunasterisk.music_67.R
+import vn.sunasterisk.music_67.base.BaseActivity
+import vn.sunasterisk.music_67.data.model.Track
+import vn.sunasterisk.music_67.data.source.TracksRepository
+import vn.sunasterisk.music_67.data.source.local.LocalDataSource
+import vn.sunasterisk.music_67.data.source.remote.RemoteDataSource
+import vn.sunasterisk.music_67.service.PlayingSet
+import vn.sunasterisk.music_67.ui.bottomsheetfragment.MenuFragment
+import vn.sunasterisk.music_67.ui.detailplaying.DetailPlayingActivity
+import vn.sunasterisk.music_67.utils.TrackAttributes
+
+const val ACTION_SEARCH_FROM_HOME = "vn.sunasterisk.music_67.ACTION_SEARCH_FROM_HOME"
+const val ACTION_SEARCH_FROM_NOW = "vn.sunasterisk.music_67.ACTION_SEARCH_FROM_NOW"
```

Comment:
2 cái search này khác gì nhau e

Reply:
1 cái search toàn bộ, 1 cái search ở cái Playing Set của em anh ơi